docs/en_US/appstore.md

```diff
@@ -5,14 +5,14 @@ lang: en_US
 
 # Appstore :shopping_cart:
 
-The Appstore is one of the central elements of LSS Manager. Here you can (de-)activate each module individually. A brief description of each module is also provided directly in the appstore.
-So that you can easily get a deeper insight into the module without having to activate the module first, the wiki entry of each module is accessible via the small blue `i`.
+The Appstore is a key element of LSS Manager. Within it, you can activate or de-activate modules individually. A brief description of each module will also be provided in the appstore. Via the small blue `i`, you can open the wiki page directly in game. 
 
 ::: tip Changes
-If you have unsaved changes, you cannot close the Appstore, a small hint message will appear.
+You cannot close the appstore with unsaved changes
 :::
-::: warning Support of the map type "MapKit"
-There are a few modules that are not supported when MapKit is selected as the map type. This is because some functionality is not provided by MapKit itself. A complete list of these modules can be found at [ℹ️ General information on modules](apps.md).
+
+::: warning Support of "MapKit" map type.
+There are a few modules which do not support MapKit map type. This is because some functionality is not provided by MapKit itself. A complete list of these modules can be found at [ℹ️ General information on modules](apps.md).
 
 For the same reason, there are also some minor restrictions for MapKit users.
```

Comment:
Could we please add that these are restrictions in the game itself? That isn't clear currently.

Reply:
see 06b03e8.
Is that up to standard? I'm not entirely sure how to make it more clear.

Reply:
The updated version should be fine 👍